types/stripe/index.d.ts

```diff
@@ -5382,18 +5518,18 @@ declare namespace Stripe {
              * Once entirely refunded, a charge can't be refunded again.
              * This method will throw an error when called on an already-refunded charge, or when trying to refund more money than is left on a charge.
              */
-            create(data: refunds.IRefundCreationOptions, options: HeaderOptions, response?: IResponseFn<refunds.IRefund>): Promise<refunds.IRefund>;
-            create(data: refunds.IRefundCreationOptions, response?: IResponseFn<refunds.IRefund>): Promise<refunds.IRefund>;
-            create(options: HeaderOptions, response?: IResponseFn<refunds.IRefund>): Promise<refunds.IRefund>;
-            create(response?: IResponseFn<refunds.IRefund>): Promise<refunds.IRefund>;
+            create?(data: refunds.IRefundCreationOptions, options: HeaderOptions, response?: IResponseFn<refunds.IRefund>): Promise<refunds.IRefund>;
```

Comment:
Why have these been made optional?

Reply:
Well, this is primarily a data model object. For some reason when I actually use the library I've noticed the quite often I don't really have these functions available. This is why I did that.